### PR TITLE
Update Regular Week2 Result & Week3 Schedule For `GBC 2022 Spring`

### PR DESCRIPTION
**Pull request by RushFTK**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by RushFTK** [comment]
Since Week2 has already ended, this PR is changed to update the week3 schedule.

**Comment by Walavouchey** [comment]
is this good to go?

**Comment by RushFTK** [comment]
@Walavouchey now is ready for merge.

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 1]
```diff
@@ -86,34 +87,63 @@ The GB Cup 2022 Spring was run by the GB Team and various community members.
...
+| 金戈铁马的鲜丰水果摊 | 货拉拉拉不拉拉布拉多 | Apr 24 (Sun) 20:00 (UTC+8) | ¹ |
+
+¹ Round 3 High Matches (Teams with 2-0 record)\
+² Round 3 Mid Matches (Teams with 1-1 record)\
+³ Round 3 Low Matches (Teams with 0-2 record)
 
 ## Mappools
 
-### Regular stage
+### Regular stage Week 3
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:121]:
```suggestion
### Regular stage week 3
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 2]
```diff
@@ -305,7 +356,7 @@ Sunday, April 10, 2022
 ### Mappool instructions
 
 1. There will be one mappool for each stage.
-   - The whole Regular stage will be seen as a single stage, meaning the same mappool will be used in all matches in the Regular stage.
+   - The Regular stage Round 1 & Round 2 will be seen as a single stage, meaning the same mappool will be used in all matches in the Round 1 and Round 2.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:359]:
```suggestion
   - Regular stage Round 1 and Round 2 are seen as a single stage, meaning they use the same mappool.
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 3]
```diff
@@ -86,34 +87,63 @@ The GB Cup 2022 Spring was run by the GB Team and various community members.
...
+¹ Round 3 High Matches (Teams with 2-0 record)\
+² Round 3 Mid Matches (Teams with 1-1 record)\
+³ Round 3 Low Matches (Teams with 0-2 record)
 
 ## Mappools
 
-### Regular stage
+### Regular stage Week 3
+
+**[Download the mappack here (116.38 MB)](https://drive.google.com/file/d/1bB4ybEiRhBWacF1tZnQGbfBu4CiRhpGl/)**
```
Comment (RushFTK) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:123]:
```suggestion
**[Download the mappack here (116.38 MB)](https://drive.google.com/file/d/1bB4ybEiRhBWacF1tZnQGbfBu4CiRhpGl/view)**
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 4]
```diff
@@ -259,6 +309,7 @@ Sunday, April 10, 2022
    - Most matches won
    - Head-to-head result (if possible)
    - Highest Buchholz score
+   - Highest Beatmap difference
```
[the remark this answers is not in the document]
Comment (RushFTK) [reply wiki/Tournaments/GBC/GBC_2022_Spring/en.md:312]: highest `beatmap wins - beatmap loses`. (result table `BD`)

[the remark this answers is not in the document]
Comment (RushFTK) [reply wiki/Tournaments/GBC/GBC_2022_Spring/en.md]: (reopen incase you might still have question)

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 5]
```diff
@@ -259,6 +309,7 @@ Sunday, April 10, 2022
    - Most matches won
    - Head-to-head result (if possible)
    - Highest Buchholz score
+   - Highest beatmap difference
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:312]:
```suggestion
   - Highest beatmap difference (`beatmap wins - beatmap losses`)
```

File: wiki/Tournaments/GBC/GBC_2022_Spring/en.md [hunk 6]
```diff
@@ -140,22 +170,22 @@ The GB Cup 2022 Spring was run by the GB Team and various community members.
 
 | #¹ | Team | Matches | Beatmaps | BD² | BU³ | Round1 | Round2 | Round3 |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2022_Spring/en.md:171]:
```suggestion
| #¹ | Team | Matches | Beatmaps | BD² | BU³ | Round 1 | Round 2 | Round 3 |
```

**Pull request merged**